torch/ao/quantization/fx/utils.py

```diff
@@ -945,3 +953,50 @@ def find_patterns(
         new_input = first_tuple.args[0][last_getitem_index]  # type: ignore[index]
         for user in list(last_getitem.users.keys()):
             user.replace_input_with(last_getitem, new_input)
+
+def _maybe_validate_activation_post_process_dtype_constraints(
+        activation_post_process: Union[ObserverBase, FakeQuantize, None],
+        dtype_with_constraints: Optional[DTypeWithConstraints]):
```

Comment:
is this still a `Optional[DTypeWithConstraints]` or just `DTypeWithConstraints`?